client/src/redux/ga-saga.js

```diff
@@ -1,9 +1,39 @@
-import { takeEvery, call, all } from 'redux-saga/effects';
+/* eslint-disable camelcase */
+import { takeEvery, call, all, select } from 'redux-saga/effects';
+import { ABTestConfig } from '../../../config/donation-settings';
```

Comment:
```suggestion
import { aBTestConfig } from '../../../config/donation-settings';
```
Nitpicky, so feel free to ignore